### DISCUSSION TITLE
Nod Ubuntu on Raspberry PI

### DISCUSSION THREAD

**Discussion by dan-mih** [CoopNetwork/GMD-blockchain#14] [Q&A]
Salut,

Sunt nou pe aici, dar am cateva intrebari:
- am ridicat un Nod Ubuntu pe Raspberry PI si ma interesa care sunt pasii urmatori. Nodul este vizibil la adresa: http://46.97.196.205:6876;
- Ma intereseaza sa validez nodul, sa fac forging etc;
Multumesc,
Dan

**Comment by georgeasiminei** [comment]
Nu iti recomand sa expui la internet un nod http (nu https) si mai ales, cel mai important, sa nu introduci parola ta privata in acel nod.

Recomandarea este sa il folosesti doar de la tine din reteaua privata. Odata ce l-ai scos de pe internet, poti sa pornesti fogingul daca ai suficiente monede in wallet.

**Reply by 888me** [reply]
Salut,

La adresa https://thecoopnetwork.io/blockchain/ sunt expuse opt noduri publice. Nodurile care vor fi hallmarked vor putea fi accesate din internet ? 
Cele opt noduri publice sunt accesibile prin https: https://node1.thecoopnetwork.io/index.html
Care este procedura de securizare  cu certificat SSL a unui nod hallmarked ?
Nodurile care obtin hallmark se vor lista pe pagina proiectului ? 
Daca nodul nu este public ("Recomandarea este sa il folosesti doar de la tine din reteaua privata.")  va putea sa faca validari ?
In momentul de fata prin leasing se poate obtine ~43% pe an. Prin rularea unui nod hallmarked se poate depasi acest procent ?

**Reply by georgeasiminei** [reply]
Salut @888me ,

Am scris aici cateva clarificari despre ce inseamna hallmark: https://github.com/CoopNetwork/GMD-blockchain/discussions/21
Faptul ca un nod este hallmarked, nu are nicio legatura cu securizarea API-ului. Pentru a-l securiza iti recomand sa folosesti un reverse proxy, cum ar fi nginx. Detalii despre cum poti configura nginx, gasesti aici https://docs.nginx.com/nginx/admin-guide/security-controls/securing-http-traffic-upstream/
